pkg/cloud/rgraph/rnode/backendservice/builder.go

```diff
@@ -23,11 +23,14 @@ import (
 	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud"
 	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/api"
 	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/rgraph/rnode"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/rgraph/rnode/all"
 	alpha "google.golang.org/api/compute/v0.alpha"
 	beta "google.golang.org/api/compute/v0.beta"
 	"google.golang.org/api/compute/v1"
 )
 
+func init() { all.RegisterBuilder("backendServices", NewBuilder) }
```

Comment:
please extract "backendServices" to a const so we can use the value from the package and make sure that no typo will be introduced in  `NewBuilderByID`. Please apply this comment to all cases.